src/Microsoft.Extensions.Logging.AzureWebApps/Internal/WebAppsLogConfiguration.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.AzureWebApps.Internal
+{
+
+    /// <summary>
+    /// Represents the configuration of the logger from Azure WebApps
+    /// </summary>
+    public class WebAppsLogConfiguration
+    {
+        /// <summary>
+        /// The configuration that has all loggers disabled 
+        /// </summary>
+        public static WebAppsLogConfiguration Disabled => new WebAppsLogConfiguration(false, false, LogLevel.None, null, false, LogLevel.None, null);
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="WebAppsLogConfiguration"/> class
+        /// </summary>
+        /// <param name="fileLoggingEnabled">True if file logging is enabled, false otherwise</param>
+        /// <param name="fileLoggingLevel">The logging level for the file logger</param>
+        /// <param name="fileLoggingFolder">The folder where the file logger stores the logs</param>
+        /// <param name="blobLoggingEnabled">True if blob logging is enabled, false otherwise</param>
+        /// <param name="blobLoggingLevel">The logging level for the blob logger</param>
+        /// <param name="blobContainerUrl">The SAS endpoint where blob logs are stored</param>
+        public WebAppsLogConfiguration(
+            bool fileLoggingEnabled,
+            LogLevel fileLoggingLevel,
+            string fileLoggingFolder,
+            bool blobLoggingEnabled,
+            LogLevel blobLoggingLevel,
+            string blobContainerUrl)
+            : this(true, fileLoggingEnabled, fileLoggingLevel, fileLoggingFolder, blobLoggingEnabled, blobLoggingLevel, blobContainerUrl)
+        {
+        }
+
+        private WebAppsLogConfiguration(
+            bool isRunningInWebApps, 
+            bool fileLoggingEnabled,
+            LogLevel fileLoggingLevel, 
+            string fileLoggingFolder,
+            bool blobLoggingEnabled, 
+            LogLevel blobLoggingLevel, 
+            string blobContainerUrl)
+        {
+            IsRunningInWebApps = isRunningInWebApps;
+
+            FileLoggingEnabled = fileLoggingEnabled;
```

Comment:
Should these be pluggable instead? for example, since we support structured logging one could log to Azure's DocumentDb (example: PackageSearch does this). Also not sure why we are wrapping around Serilog when an end user can directly use it...will talk offline...